test/spec/repeat.spec.js

```diff
@@ -401,19 +410,24 @@ describe('repeat functionality', () => {
                 <rep>
                     <txt/>
                     <num>5</num>
+                    <is-num-relevant>yes</is-num-relevant>
```

Comment:
Were these perhaps changes that were never used? Or just wanted to exactly match https://github.com/enketo/enketo-core/pull/870/files#diff-f2896b2fc4580368e34c58eec902cffe5d8e202a73f906034a965b8be7c6ebaaR875?

Reply:
Missed when extracting changes into new forms.